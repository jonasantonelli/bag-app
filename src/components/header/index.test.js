import Header from '.';
import mockDatabase from '../../../server/database';

describe('header component', () => {
    it('it renders correctly', () => {
        const products = mockDatabase.bag[0].items;
        expect(Header(products)).toMatchSnapshot();
    });
});
