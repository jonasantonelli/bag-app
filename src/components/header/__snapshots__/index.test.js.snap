// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header component it renders correctly 1`] = `

<header class="header">
  <h1>
    My Bag
  </h1>
  <div id="render-total-price">
    <div class="price">
      Total: â‚¬ 2,244.05
    </div>
  </div>
</header>

`;
